config/routes.rb

```diff
@@ -1,3 +1,6 @@
+require 'sidekiq/web'
+Sidekiq::Web.set :sessions, false
```

Comment:
I think you mentioned this to me separately but since it's not in the docs - could you explain this line?

Reply:
I verified the functionality of this PR, but I am also curious about what these lines do and about the answer to the nginx question.

Reply:
We don't need sidekiq to handle sessions, since we do that in the app itself already.